fr.yml

```diff
@@ -1,57 +1,57 @@
 setup:
-  ^SetupCaption: "Installer $(^Name) v${VERSION}${VERSION_NOTE}"
-  ^UninstallCaption: "Désinstaller $(^Name) v${VERSION}${VERSION_NOTE}"
-  SECTION_MAIN: "$(^NameDA) (requis)"
-  SECTION_STARTMENU: "Raccourcis Menu Démarrer"
-  SECTION_DESKTOP: "Raccourci Bureau"
-  INSTALLER_RUNNING: "Une installation de $(^Name) est déjà en cours"
-  NON_EMPTY_FOLDER_ERROR: "$(^Name) doit être installé dans un dossier vide"
-  OPEN_PROGRAM: "Ouvrir $(^NameDA)"
+  ^SetupCaption:  "Installation de $(^Name) v${VERSION}${VERSION_NOTE}"
+  ^UninstallCaption:  "Désinstallation de $(^Name) v${VERSION}${VERSION_NOTE}"
+  SECTION_MAIN:  "$(^NameDA) (requis)"
+  SECTION_STARTMENU: "Raccourcis du menu Démarrer"
+  SECTION_DESKTOP: "Raccourci sur le bureau"
+  INSTALLER_RUNNING: "L'installation de $(^Name) est déjà en cours"
+  NON_EMPTY_FOLDER_ERROR: "$(^Name) ne peut pas être installé dans un dossier non vide"
+  OPEN_PROGRAM: "&Ouvrir $(^NameDA)"
   CLEANCONFIRM: "&Supprimer les paramètres du programme"
   INSTALLED_HEADER: "Déjà installé"
-  INSTALLED_TEXT: "Sélectionnez l'action à effectuer"
-  INSTALLED_SAME: "$(^NameDA) v${VERSION} est déjà installé. Sélectionnez l'action que vous voulez effectuer."
-  INSTALLED_OLD: "Une ancienne version de $(^NameDA) est installée sur votre système. Sélectionnez l'action que vous voulez effectuer."
-  INSTALLED_NEW: "Une version plus récente de $(^NameDA) est déjà installée ! Il n'est pas recommandé d'installer une version plus ancienne. Si vous voulez réellement installer cette version plus ancienne, il est recommandé de désinstaller en premier la version actuelle. Sélectionnez l'action que vous voulez effectuer."
-  REINSTALL: "&Réinstaller"
+  INSTALLED_TEXT: "Sélectionnez l'opération à effectuer."
+  INSTALLED_SAME: "$(^NameDA) v${VERSION} est déjà installé.  Sélectionnez l'opération que vous souhaitez effectuer."
+  INSTALLED_OLD: "Une version plus ancienne de $(^NameDA) est installée sur votre système. Sélectionnez l'opération que vous souhaitez effectuer."
+  INSTALLED_NEW: "Une version plus récente de $(^NameDA) est déjà installée !  Il n'est pas recommandé d'installer une version plus ancienne. Si vous souhaitez vraiment installer cette version plus ancienne, il est préférable de désinstaller d'abord la version actuelle."
```

Comment:
The final sentence was removed.
- English: "...uninstall the current version first. **Select the operation you want to perform.**"
- French: "...désinstaller d'abord la version actuelle."